includes/widgets/calendar.php

```diff
@@ -137,7 +137,11 @@ public function form( $instance ) {
 				       class="widefat simcal-field simcal-field-standard simcal-field-text"
 				       value="<?php echo $title; ?>">
 			</p>
-
+				<?php 
+				if ( defined( 'REST_REQUEST' ) && REST_REQUEST ) {
+					$this->calendars = simcal_get_calendars();
+				}
```

Comment:
Can this be moved to line 127?
Also, should we check if `$this->calendars` is not defined?

Reply:
Move to 127. 
We have already define "public $calendars = array();" in class at line 31.

Reply:
> We have already define "public $calendars = array();" in class at line 31.

What I meant was do you want to populate `$this->calendars` at all times even if it already has value?

Reply:
added condition to check if it is empty.